client/login/wp-login/style.scss

```diff
@@ -191,7 +192,8 @@ $image-height: 47px;
 	.wp-login__links a,
 	.logged-out-form__links a,
 	.logged-out-form__link-item,
-	.translator-invite__content a {
+	.translator-invite__content a,
+  	.magic-login__footer a {
```

Comment:
Small fix needed; there's a mixture of tabs and spaces here.

Reply:
Fixed the formatting of that file 👍